### DISCUSSION TITLE
Can We Build Release Version of .aab or .apk

### DISCUSSION THREAD

**Discussion by ajithandro** [opengisch/QField#4997] [Q&A]
### Release Version Application and Deployment

1. We can build a **Release Version** of the **.aab** or **.apk** file of QField with **Android Studio** or **Visual Studio?**
2. How we can create/set new keystore to create Release Version of Application
3. Seems the QField docker build process only gives the debug version of the QField Application alone
`triplet=arm64-android ./scripts/build.sh`

How can we generate/build applications with the build type of release/build? in QField

Once the Build is successful, I can get this Statement alone
```
BUILD SUCCESSFUL in 5s
33 actionable tasks: 3 executed, 30 up-to-date
Android package built successfully in 43.773 ms.
  **-- File: /usr/src/qfield/build-arm64-android/src/app/android-build//build/outputs/apk/debug/android-build-debug.apk**
CPack: - package: /usr/src/qfield/build-arm64-android/Kcube Field-0.1.1-Android.json generated.
```
 In this Path Debug Build Only Exists 

Thanks

**Comment by m-kuhn** [comment]
For reference:

> the best reference for this is the current release pipeline
> 
> https://github.com/opengisch/QField/blob/c9ead83/.github/workflows/android.yml#L174
> 
> any pull request to improve the documentation will be very welcome

See https://github.com/opengisch/QField/issues/4962#issuecomment-1900619182
